dll/win32/url/url_main.c

```diff
@@ -105,7 +105,12 @@ void WINAPI FileProtocolHandlerA(HWND hWnd, HINSTANCE hInst, LPCSTR pszUrl, int
  */
 void WINAPI OpenURLA(HWND hwnd, HINSTANCE inst, LPCSTR cmdline, INT show)
 {
+#ifdef __REACTOS__
+    TRACE("(%p, %p, %s, %d)\n", hwnd, inst, debugstr_a(cmdline), show);
+    ShellExecuteA(hwnd, NULL, cmdline, NULL, NULL, show);
+#else
     FIXME("(%p, %p, %s, %d): stub!\n", hwnd, inst, debugstr_a(cmdline), show);
+#endif
```

Comment:
It might be worth to split off the changes to `dll/win32/url/url_main.c` into a separate PR, as I think that part is much less controversial than the rest of this PR. It might be possible to get approval for that part much easier than for the rest of this PR. E.g. Personally I would approve the url_main part right away.
The rest of this PR may be a bit controversial for some of us because it couples the mshtml/ie part more tightly to the shell, which is something many of the devs tried to avoid and commonly agreed to, some years ago.

Reply:
Fixed in `master`.

Reply:
Can be stripped now in here, because got merged already.

Reply:
Deleted change.